README.md

```diff
@@ -56,20 +62,18 @@ var Component = React.createClass({
     onoutsideevent
   ],
 
-  handleOutsideEvent: function(evt) {
-    // ...handling code goes here...
-  }
+  ...
 });
 ```
 
-## Regulate whether or not to listen for outside events
+## Regulate whether or not to listen for events
 
 When using this mixin, a component has two functions that can be used to explicitly listen for, or do nothing with, outside events
 
-- `enableOnOutsideEvent()` - Enables outside click listening by setting up the event listening bindings.
-- `disableOnOutsideEvent()` - Disables outside click listening by explicitly removing the event listening bindings.
+- `enableOnOutsideEvents(evtType)` - Enables outside event listening on a specific event type by setting up the event listening bindings. Defaults `evtType` to the `listenToOutsideEvent` prop.
```

Comment:
replace "prop" with "property", and then it's probably a good idea if we also add "or `mousedown` is no such property is set." at the end, since we have a default binding